frontend/src/modules/editor/contentObject/views/editorMenuView.js

```diff
@@ -114,28 +129,31 @@ define(function(require){
     */
     getItemHeirarchy: function(model, done) {
       var hierarchy = [];
-      if(model.get('_type') === 'menu') {
+      if (model.get('_type') === 'menu') {
         hierarchy.push(model);
       }
       var __this = this;
       var _getParent = function(model, callback) {
         var parent = __this.contentobjects.findWhere({ _id: model.get('_parentId') });
-        if(parent) {
+        if (parent) {
           hierarchy.push(parent);
           return _getParent(parent, callback);
         }
         hierarchy.push(Origin.editor.data.course);
         callback();
       };
       _getParent(model, function() {
-        if(typeof done === 'function') done.call(__this, hierarchy.reverse());
+        if (typeof done === 'function') done.call(__this, hierarchy.reverse());
       });
     },
 
     onSelectedItemChanged: function(model) {
-      if(model.get('_id') === Origin.editor.currentContentObject && Origin.editor.currentContentObject.get('_id')) {
-        return;
+      if (model) {
```

Comment:
This can probably just be `if(model && model.get('_id') ...`, or
```js
if (!model || !Origin.editor.currentContentObject) {
  return;
}
if(model.get('_id') === Origin.editor.currentContentObject.get('_id')) {
   return;
}
```
Prefer the first for brevity, although is a bit long